girder/stemserver_plugin/models/stemimage.py

```diff
@@ -326,3 +344,11 @@ def _get_dataset_in_chunks(self, dataset, max_chunk_size=64000):
             total_read += read_size
 
             yield array
+
+    def _get_path_to_type(self, type):
+        """Get the path to the dataset for the given type"""
+        if type == 'electron':
+            return '/electron_events/frames'
+        elif type == 'raw':
+            return '/stem/frames'
```

Comment:
Looking at these together, I don't think we want 'stem' in the frames path ( the frames could be from another detector type ). So I think we should have `/electron_events` and `/frames`.

Reply:
Done. Let me know if there's something I should do about `GET /stem_images/{id}/frames` timing out for raw data. Otherwise, I think we are good to go.

Reply:
Can we provide an `offset` and `limit` parameter, so the client could stream in chunks?

Reply:
@cjh1 I added it. I think it works. Can you check to make sure it looks accurate, and let me know if you can think of a way to make this simpler/shorter?

Reply:
I did a couple of tests with it and it looks like it works as intended.